plugins/woocommerce-admin/client/wp-admin-scripts/tags-tracking/index.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { recordEvent } from '@woocommerce/tracks';
+
+const addNewAttribute = document.querySelector( '#addtag #submit' );
```

Comment:
Could we rename this for `addNewTag`?

Reply:
I updated this here: https://github.com/woocommerce/woocommerce/pull/33118/commits/13bd524a02368c29c0e1cffb6adafe25bd9f8af1